node/resolvers/Mutations/Users.ts

```diff
@@ -332,14 +363,18 @@ export const deleteUser = async (_: any, params: any, ctx: Context) => {
 
 export const impersonateUser = async (_: any, params: any, ctx: Context) => {
   const {
-    clients: { session },
     vtex: { logger, sessionToken },
   } = ctx
 
   const { userId } = params
 
   try {
-    await session.updateSession('impersonate', userId, [], sessionToken)
+    setChangeSession({
```

Comment:
There is a sonarcloud issue on these calls to `setChangeSession`:
```
Consider using 'await' for the promise inside this 'try' or replace it with 'Promise.prototype.catch(...)' usage.
```
Is there a problem if we add `await` on it? (the same for all other calls of this function)